packages/app/src/components/contract/ContractInfoTab.vue

```diff
@@ -237,13 +243,15 @@ const readProxyFunctions = computed(() => {
 const tabs = computed(() => {
   const isVerified = !!props.contract?.verificationInfo;
   const isProxy = !!props.contract?.proxyInfo;
+  const isDiamondProxy = !!props.contract?.diamondProxyInfo;
   if (isVerified || isProxy) {
```

Comment:
`isDiamondProxy` has to be added to the `if` condition. Otherwise the line added below is unreachable.